ruby-focal/3.0.x/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM socrata/base-focal
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+# skip installing gem documentation
+RUN mkdir -p /usr/local/etc \
+	&& { \
+		echo 'install: --no-document'; \
+		echo 'update: --no-document'; \
+	} >> /usr/local/etc/gemrc
+
+ENV RUBY_MAJOR 3.0
```

Comment:
Copied from ruby-focal/3.1.x -- I only changed the ruby major version on lines 11 and line 41